onnxruntime/core/providers/cpu/tensor/upsample.h

```diff
@@ -79,7 +79,7 @@ class UpsampleBase {
       const Tensor* scale;
       bool get_scale = info.TryGetConstantInput(scales_input_idx_, &scale);
```

Comment:
Side not: This should be updated at some point to use the graph_utils to get constant input in case the initializer is in a parent graph.